tests/stop_condition_eventually/Makefile

```diff
@@ -0,0 +1,54 @@
+SPIN?=spin
+CC=gcc
+CFLAGS=-O2
+
+# We are testing the sources, not the installed application
+SDL2PROMELA=PYTHONPATH=../../ python3 -m sdl2promela.sdl2promela
+
+RES_DIR=resources
+
+TEST_DIR=out
+
+CREATE_EMPTY_TEST_DIR=rm -r -f ${TEST_DIR} && mkdir -p ${TEST_DIR}
+DELETE_TEST_DIR=rm -r ${TEST_DIR}
+
+TESTS = \
+	test_stop_condition_eventually
+
+.SILENT : \
+	${TESTS}
+
+.PHONY : \
+	${TESTS} \
+	check
+
+check: ${TESTS}
+
+##############################INTEGRATION TESTS################################
+
+# These tests are created based on the tests supplied with custom Spin version
+# used by N7 Space (see github.com/n7space/Spin). Their purpose is to validate
+# whether the generated Promela code actually works inside Spin.
+# As sdl2promela generates only inlines (representing the SDL logic) which need
+# external data and process definitions (similarly to opengeode generating
+# only C functions, which need type declarations generated by asn1scc and
+# glue code generated by Kazoo), they are wrapped inside "host" Promela
+# files
+
+test_stop_condition_eventually:
+	echo "Test stop_condition_eventually"
+	${CREATE_EMPTY_TEST_DIR}
+	${SDL2PROMELA} -o ${TEST_DIR}/controller.pml ${RES_DIR}/controller/system_structure.pr ${RES_DIR}/controller/controller.pr
+	${SDL2PROMELA} -o ${TEST_DIR}/actuator.pml ${RES_DIR}/actuator/system_structure.pr ${RES_DIR}/actuator/actuator.pr
+	${SDL2PROMELA} -o ${TEST_DIR}/scl.pml --scl ${RES_DIR}/scl.txt
+	cp ${RES_DIR}/dataview.pml ${TEST_DIR}/dataview.pml
```

Comment:
Also, I remember that my original comment was about eventually.... but this MR also introduces filter out. Is it tested?

Reply:
No, It wasn't. I've added a test for filter_out.